pipelines/apCcdVisitQualityCore.yaml

```diff
@@ -0,0 +1,16 @@
+description: |
+  Tier 1 AP plots and metrics to assess CcdVisit quality
+tasks:
+  analyzeDiaSourceTableCore:
+    class: lsst.analysis.tools.tasks.DiaSourceTableCcdVisitAnalysisTask
+    config:
+      # Add plots and metrics that work when we have some
+      # plots.label: APlotClass
+      metrics.numSsObjects: NumSsObjectsMetric
+      connections.outputName: diaSourceTableCore
+      #metrics.skyFluxVisitStatisticMetric: SkyFluxStatisticMetric
+      #metrics.skyFluxVisitStatisticMetric.applyContext: VisitContext
+      python: |
+        from lsst.analysis.tools.analysisPlots import *
+        from lsst.analysis.tools.analysisMetrics import *
+        from lsst.analysis.tools.contexts import *
```

Comment:
You don't need all of these imports unless you are uncommenting some of the lines.